content/cloud-docs/api-docs/providers.mdx

```diff
@@ -254,79 +312,411 @@ curl \
+            },
+            {
+              "id": "provpltfrm-A1PHitiM2KkKpVoM",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-BLJWvWyJ2QMs525k",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-qQYosUguetYtXGzJ",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-pjDHFN46y193bS7t",
+              "type": "registry-provider-platforms"
+            }
+          ],
+          "links": {
+            "related": "/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms"
+          }
+        }
+      },
+      "links": {
+        "shasums-download": "https://archivist.terraform.io/v1/object/dmF1b...",
+        "shasums-sig-download": "https://archivist.terraform.io/v1/object/dmF1b..."
+      }
+    }
+  ],
+  "links": {
+    "self": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions?page%5Bnumber%5D=1&page%5Bsize%5D=20",
+    "first": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions?page%5Bnumber%5D=1&page%5Bsize%5D=20",
+    "prev": null,
+    "next": null,
+    "last": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions?page%5Bnumber%5D=1&page%5Bsize%5D=20"
+  },
+  "meta": {
+    "pagination": {
+      "current-page": 1,
+      "page-size": 20,
+      "prev-page": null,
+      "next-page": null,
+      "total-pages": 1,
+      "total-count": 1
+    }
+  }
+}
+```
+### Note
+
+`shasums-uploaded` and `shasums-sig-uploaded` may be false if those files haven't been uploaded to Archivist yet. In this case, instead of including links to `shasums-download` and `shasums-sig-download`, the response will include upload links (`shasums-upload` and `shasums-sig-upload`) instead.
+
+## Get Version (private provider)
+
+`GET /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name/versions/:version`
+
+### Parameters
+
+| Parameter            | Description                                                                                              |
+| -------------------- | -------------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the provider belongs to.                                                    |
+| `:namespace`         | The namespace of the provider.                                                                           |
+| `:name`              | The provider name.                                                                                       |
+| `:version`           | The provider version of the provider for which the platform is being created                             |
```

Comment:
This sentence is a little unclear because there can be multiple platforms per version.

I'd suggest something like: 
"The version of the provider being created to which different platforms can be added"

Reply:
haha thank you was a bit brain dead at this point